src/components/snackbar/examples/snackbar-positioning.tsx

```diff
@@ -0,0 +1,56 @@
+import { Component, Element, h } from '@stencil/core';
+
+/**
+ * Positioning on large and small screens
+ *
+ * Snackbars are by default center-aligned and placed at the bottom of the screen.
+ * However, on larger screens, they can optionally be displayed on the leading edge
+ * which would be the left side in LTR languages, or the right side in RTL ones.
+ *
+ * To do so, you can take advantage of the provided CSS variables,
+ * and keep in mind that the the Snackbar uses `position: fixed;`
+ * to determine its location.
+ *
+ * :::tip
+ * When customizing the Snackbars for usage in progressive web applications,
+ * remember to consider the safe areas, and add the
+ * [environment variables](https://developer.mozilla.org/en-US/docs/Web/CSS/env)
+ * in in your calculations.
+ *
+ * For example: `--snackbar-bottom: env(safe-area-inset-left, 0)`.
+ * :::
+ */
+@Component({
+    tag: 'limel-example-snackbar-positioning',
+    shadow: true,
+    styleUrl: 'snackbar-positioning.scss',
+})
+export class SnackbarPositioningExample {
+    @Element()
+    private host: HTMLLimelExampleSnackbarElement;
+
+    private triggerSnackbarWithoutAction: (event: MouseEvent) => void;
+
+    constructor() {
```

Comment:
I wonder if we should use `connectedCallback` instead, knowing that the appers tend to copy code from the documentation and it might be more "correct". I was surprised this works 😅, as `constructor` could technically be called before we attach the snackbar to the DOM.

Reply:
I take it back, I looked through the source and it seems this is the way :)